src/lib/ui/thermostat/ThermostatCard.tsx

```diff
@@ -0,0 +1,149 @@
+import classNames from 'classnames'
+import { useState } from 'react'
+import {
+  isThermostatDevice,
+  type ThermostatDevice,
+  type ThermostatDeviceProperties,
+} from 'seamapi'
+
+import { FanIcon } from 'lib/icons/Fan.js'
+import { OffIcon } from 'lib/icons/Off.js'
+import { DeviceImage } from 'lib/ui/device/DeviceImage.js'
+import { ClimateSettingStatus } from 'lib/ui/thermostat/ClimateSettingStatus.js'
+
+interface ThermostatCardProps {
+  device: ThermostatDevice
+}
+
+export function ThermostatCard({ device }: ThermostatCardProps): JSX.Element {
+  return (
+    <div className='seam-thermostat-card'>
+      <Content device={device} />
+    </div>
+  )
+}
+
+function Content(props: { device: ThermostatDevice }): JSX.Element | null {
+  const { device } = props
+
+  const [temperatureUnit, setTemperatureUnit] = useState<
+    'fahrenheit' | 'celsius'
+  >('fahrenheit')
+
+  const toggleTemperatureScale = (): void => {
+    setTemperatureUnit(
+      temperatureUnit === 'fahrenheit' ? 'celsius' : 'fahrenheit'
+    )
+  }
+
+  if (device == null || (device != null && !isThermostatDevice(device))) {
+    return null
+  }
+
```

Comment:
Do we need this check? It should be handled already by the types / parent.
```suggestion
```

Reply:
Oops, that's old—thanks!